src/Components/Components/src/RenderHandle.cs

```diff
@@ -44,6 +44,11 @@ public Dispatcher Dispatcher
         public bool IsInitialized
             => _renderer != null;
 
+        /// <summary>
+        /// Gets a value that determines if the <see cref="Renderer"/> is triggering a render in response to a hot-reload change.
+        /// </summary>
+        public bool IsHotReloading => _renderer.IsHotReloading;
```

Comment:
Does this also need a `null` check as in the other methods?
```suggestion
        public bool IsHotReloading => _renderer != null && _renderer.IsHotReloading;
```

Reply:
Good catch! I updated the nullability for `_renderer` along with this change.